samcli/cli/cli_config_file.py

```diff
@@ -15,10 +15,15 @@
 from samcli.lib.config.exceptions import SamConfigVersionException
 from samcli.cli.context import get_cmd_names
 from samcli.lib.config.samconfig import SamConfig, DEFAULT_ENV
+from samcli.lib.utils.sam_logging import SamCliLogger
 
 __all__ = ("TomlProvider", "configuration_option", "get_ctx_defaults")
 
+# Adding Samconfig prefix to the formatter here to show explicitly that the log messages generated
+# are exclusively from the configuration file context.
+
 LOG = logging.getLogger(__name__)
+SamCliLogger.configure_logger(LOG, logging.Formatter("Samconfig: %(message)s"), logging.INFO)
```

Comment:
What does this do? Modify log format for every log or just of this file?

Reply:
Ohhh... never mind it is accept LOG object, so probably just for this file.

Reply:
There is no logger configured at this point, because the configuration file decorator is called in before https://github.com/awslabs/aws-sam-cli/blob/develop/samcli/cli/main.py#L97